tests/test_integrity_checker.py

```diff
@@ -644,3 +647,80 @@ def test_global_noqa_works_for_syntax_errors(self):
         ])
         for variant in ['# noqa: *', '# noqa: S001', '# noqa']:
             self.has_no_errors(program.format(variant))
+
+        self.config = Configuration(
+            ignore=[],
+            message_template=None,
+            style=DocstringStyle.SPHINX,
+            raise_on_missing_docstrings=False
+        )
+
+    def test_check_for_missing_docstrings(self):
```

Comment:
This could probably be broken up into multiple unit tests.  If it's in one unit tests because of the hassle of performing configuration, etc., then you could put it into its own TestCase and have a setUp method.  If it's broken up, it will make failing tests easier to diagnose. If we somehow broke the check for missing docstrings on functions but not methods, for example, then the method's name would immediately let us know what is wrong.